package.json

```diff
@@ -105,6 +105,7 @@
         "subPermissions": [
           "settings.erm-usage.enabled",
           "aggregatorsettings.collection.get",
+          "aggregatorsettings.exportcredentials.item.get",
           "aggregatorsettings.item.get",
```

Comment:
Do we need to keep this permission for older mod-er-usage versions?
Here this permission is deleted among others:
https://github.com/folio-org/mod-erm-usage/pull/185/files

Reply:
Ok, I see, the permissions are not replaced, they are still used.

Reply:
So no objections?